cmd/aws-janitor-boskos/main.go

```diff
@@ -44,18 +51,33 @@ var (
 	sweepSleepDuration time.Duration
 	logLevel           = flag.String("log-level", "info", fmt.Sprintf("Log level is one of %v.", logrus.AllLevels))
 	dryRun             = flag.Bool("dry-run", false, "If set, don't delete any resources, only log what would be done")
+
+	instrumentationOptions prowflagutil.InstrumentationOptions
 )
 
 const (
 	sleepTime = time.Minute
 )
 
+var (
+	cleaningTimeHistogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
nit: this could probably be included in the `var` block above.

Reply:
missed the var block above 🤦